ui-speedspacechart/src/types/chartTypes.ts

```diff
@@ -1,8 +1,10 @@
+import { LINEAR_LAYERS_HEIGHTS } from '../components/const';
```

Comment:
When I build the project, this line throws an error:
```
 [plugin typescript] src/types/chartTypes.ts (1:1): @rollup/plugin-typescript TS6133: 'LINEAR_LAYERS_HEIGHTS' is declared but its value is never read.
/home/jacomyal/code/osrd-ui/ui-speedspacechart/src/types/chartTypes.ts:1:1

1 import { LINEAR_LAYERS_HEIGHTS } from '../components/const';
```

I think it is because it creates a circular dependency between `chartTypes.ts` and `const.ts`.

Reply:
Actually this wasn't used anymore so I removed it.